src/pybind/mgr/dashboard_v2/frontend/README.md

```diff
@@ -1,44 +1,48 @@
-# Openattic
+# Dashboard
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 1.6.3.
 
+## Installation
+
+Run `npm install` to install the required packages locally.
+
 ## Development server
 
-Run `ng serve` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
+Run `npm run start` for a dev server. Navigate to `http://localhost:4200/`. The app will automatically reload if you change any of the source files.
```

Comment:
You are able to run `ng serve` directly or to run `npm start`. But `npm start` is the same as `npm run start`.  So I would stick to `npm start`. As we might change the `ng serve` command in the future to use a proxy to communicate with our backend elsewhere.